src/styles/docTemplate.css

```
@@ -254,19 +256,48 @@
     }
 
     .expandable-image-modal-content {
-      padding: 0;
+      --expandable-image-modal-padding: clamp(44px, 4vw, 48px);
+      --expandable-image-modal-body-max-height: calc(
+        100vh - var(--expandable-modal-backdrop-padding) -
```

Stylelint (17.14.1)

[error] Unexpected newline after "-" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)